packages/terra-status-view/src/terra-dev-site/doc/example/ToggleVariants.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import StatusView from 'terra-status-view/lib/StatusView';
+
+class ToggleVariants extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selected: 'error',
+    };
+    this.handleOnSelect = this.handleOnSelect.bind(this);
+  }
+
+  handleOnSelect(event) {
+    this.setState({ selected: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+        <StatusView variant={this.state.selected} />
+        <fieldset>
+          <legend>Change Variant</legend>
+          <select id="statusViewVariant" name="statusViewVariant" value={this.state.selected} onChange={this.handleOnSelect}>
+            <option value="error">Error</option>
+            <option value="no-data">No Data</option>
+            <option value="no-matching-results">No Matching Results</option>
+            <option value="not-authorized">Not Authorized</option>
+          </select>
+        </fieldset>
+      </div>
+    );
+  }
+}
+
+export default ToggleVariants;
```

Comment:
If you're interested, these example components are prime candidates for the usage of [hooks](https://reactjs.org/docs/hooks-state.html).

Reply:
good idea, update to use hooks https://github.com/cerner/terra-core/pull/2453/commits/8ad0a3ff2a728771b709167dc3a0617952b5f864

Reply:
@yuderekyu +1 on the select and checkbox toggle examples. hooks were a nice touch, really shortened up the implementation.